docker-compose.yml

```diff
@@ -121,6 +121,8 @@ services:
     image: grafana/grafana:latest
     env_file:
       - ./monitoring/configuration/grafana-plugins.env
+    environment:
+      - ADMIN_PASSW=${GF_SECURITY_ADMIN_PASSWORD}
```

Comment:
Could you just set 
```
GF_SECURITY_ADMIN_USER=${GF_SECURITY_ADMIN_USER}
GF_SECURITY_ADMIN_PASSWORD=${GF_SECURITY_ADMIN_PASSWORD}
```

Reply:
done https://github.com/waku-org/waku-simulator/pull/56/commits/9f93120a631f54ecb944637da32861f8ab32da27